### PR TITLE
add my profile

### PR DESCRIPTION
**Pull request by dkennell**

### CODE REVIEW

**Comment by ivalentine** [comment]
@dkennell Hi David! I've reviewed your pull request, and it looks like you forgot to add your card to `index.html`. Once you've added your card to `index.html`, please update your pull request and we'll be happy to take another look.

If you have any questions, feel free to Ask a Question on this lesson on Learn. Thanks!

cc @aturkewi

**Comment by dkennell** [comment]
Hi, sorry about that, and thanks for letting me know. I've updated
index.html and opened up another PR. :)

David

File: index.html [hunk 1]
```diff
@@ -20,6 +20,19 @@
           <h1>Every Story Has Heroes</h1>
           <h2>We're a group of people on a journey to learn to love code.</h2>
         </div>
+        
+        <div class="roster-cards-container">
```
Comment (bhollan) [inline index.html]: There should only be one of these in the entire file.  This is not part of the "card" of each student.

File: css/styles.css [hunk 2]
```diff
@@ -133,6 +133,14 @@ a:visited {
   height: 400px;
 }
 
+#david-kennell-cover{
+  background-image: url(http://i.imgur.com/6sZfC3n.jpg?1);
```
Comment (bhollan) [inline css/styles.css]: These URLs should have `"`s around them.

**Comment by bhollan** [comment]
Hi @dkennell.

I think we're going to close this for now.  It's still here and ready to go in whenever you're ready to get the requested changes in.  Feel free to Ask A Question for help and/or re-open this PR whenever you're ready.

CC: @aturkewi

**Pull request closed without merging**